kernel/pre_env.mli

```diff
@@ -45,11 +45,15 @@ type named_context_val = private {
   env_named_map : (Context.Named.Declaration.t * lazy_val) Id.Map.t;
 }
 
+type rel_context_val = private {
+  env_rel_ctx : Context.Rel.t;
+  env_rel_map : (Context.Rel.Declaration.t * lazy_val) Range.t;
```

Comment:
What's the lazy_val for?

Reply:
It's the result of the VM compilation. Note that it was already there before, but as another field. I decided to go the same route as the fast-var-lookup branch, by merging the two environments into one.